src/BenchmarkDotNet/Portability/Cpu/ProcCpuInfoParser.cs

```diff
@@ -25,15 +33,46 @@ internal static CpuInfo ParseOutput([CanBeNull] string content)
                 if (logicalCore.TryGetValue(ProcCpuInfoKeyNames.ModelName, out string modelName))
                 {
                     processorModelNames.Add(modelName);
+                    nominalFrequency = ParseFrequencyFromBrandString(modelName);
                     logicalCoreCount++;
                 }
+                
+                if (logicalCore.TryGetValue(ProcCpuInfoKeyNames.MinFrequency, out string minCpuFreqValue)
+                    && double.TryParse(minCpuFreqValue.Replace(',','.'), out double minCpuFreq))
```

Comment:
`double.TryParse` uses the current culture by default. Thus, this construction will not work in some environments; we should always use `HostEnvironmentInfo.MainCultureInfo` for each parsing operation. To avoid copy-pasting, I suggest introducing additional methods in `Frequency.cs`:
```cs
public static bool TryParse(string s, FrequencyUnit unit, out Frequency freq)
{
    var success = double.TryParse(s, NumberStyles.Any, HostEnvironmentInfo.MainCultureInfo, out double result);
    freq = new Frequency(result, unit);
    return success;
}

public static bool TryParseHz(string s, out Frequency freq) => TryParse(s, FrequencyUnit.Hz, out freq);
public static bool TryParseKHz(string s, out Frequency freq) => TryParse(s, FrequencyUnit.KHz, out freq);
public static bool TryParseMHz(string s, out Frequency freq) => TryParse(s, FrequencyUnit.MHz, out freq);
public static bool TryParseGHz(string s, out Frequency freq) => TryParse(s, FrequencyUnit.GHz, out freq);
```
Then, we can use `Frequency.TryParseMHz` everywhere.